extensions/github-gist/src/util/utils.ts

```diff
@@ -23,13 +23,12 @@ export const imgExt = [".svg", ".gif", ".jpg", ".jpeg", ".png"];
 
 export const getGistDetailContent = (gistFile: GistItem, gistFileContent: string) => {
   const gistFileExt = parse(gistFile.filename).ext;
+
   if (imgExt.includes(gistFileExt)) {
     return `![](${gistFile.raw_url})`;
   }
-  if (gistFileExt == ".md") {
```

Comment:
@Aayush9029 
Do not delete this code, please treat the Markdown format file separately: Do not use code block syntax for Markdown format gist